python/cuspatial/setup.py

```diff
@@ -31,6 +31,8 @@
 
 cuda_include_dir = os.path.join(CUDA_HOME, "include")
 
+CUSPATIAL_ROOT = os.environ.get("CUSPATIAL_ROOT", "../../cpp/build/")
```

Comment:
I'm curious, when running a setup.py, what reason would someone have for **not** using the build in `../../cpp/build`?  Seems like that would be confusing, but maybe I'm missing an important use case.  If so, maybe add a comment?

Reply:
I don't want to make any assumptions about where the C++ build directory is. In `rapids-compose` it's `cpp/build/release` (and that's a symlink to `cpp/build/cuda-10.2/fix__cmake-paths/release`).

Most of our other scripts assume a convention that `$<PROJ>_ROOT` is the C++ build dir, so this follows that if set, and it not it uses our CI paths as the fallback path.

Reply:
If there's a convention in place then I'm fine with no comment.